### ISSUE TITLE
On giving the 'Show numbers' command, voice access is not showing numbers for 'Next/Previous' buttons present at the legend list in windows.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#24430]
**Expected result:** 
On giving the 'Show numbers' command, voice access should show numbers for 'Next/Previous' buttons present at the legend list in windows.

**Actual Result:**
On giving the 'Show numbers' command, voice access is not showing numbers for 'Next/Previous' buttons present at the legend list in windows.

**Steps to Reproduce:**

1. Open the provided test URL: https://codepen.io/Boris-Trajkovic/pen/myEQomV  in edge/chrome browser.
2. Press tab key to navigate to the chart legends.
3. Give 'show numbers' command.
4. Observe whether numbers are displayed for the "Next" and "Previous" buttons or not.

**Product version**
Highcharts Core version: 9.3.3

**Pre-requisite:**
Enable Voice Access:
Navigate to Settings > Accessibility > Speech > Voice Access
Or search "Voice Access" in Start menu

**Test Environment:**
Test URL: Highcharts
OS: Windows (with Voice Access enabled)
URL: https://codepen.io/Boris-Trajkovic/pen/myEQomV
Assistive Technology: Voice Access

**User Impact:**
Voice access users will be impacted if the "Next" and "Previous" buttons in the legends list are not assigned numbers, making it difficult to identify and activate these controls using voice commands. As a result, they cannot navigate through the legend items, preventing them from fully accessing or understanding the chart information. This creates a significant accessibility barrier for individuals who rely solely on voice input, limiting their ability to interact with the application independently and consistently.

**Refer Attachments:**

1. On giving the 'Show numbers' command, voice access is not showing numbers for 'Next/Previous' buttons present at the legend list in windows.png
2. On giving the 'Show numbers' command, voice access is not showing numbers for 'Next/Previous' buttons present at the legend list in windows.mp4

<img width="1943" height="1153" alt="Image" src="https://github.com/user-attachments/assets/fe7d5bd3-db47-4e47-b52a-16f4f33aa0cf" />
https://github.com/user-attachments/assets/7405c562-a812-4722-8a05-0e8e2c2d14ee

**Comment by magdalena-gut** [comment]
Hi @msftedad , thank you for the detailed report! I tested it with the latest version of Highcharts v12.5.0 and observed the same result. I've added this to our backlog for further investigation.

**Comment by oysteinmoseng** [comment]
If I remember correctly, these buttons are not made accessible - the keyboard navigation simply autoscrolls the list when the end of the visible range is reached. Will be fixed in the new a11y module, but keeping the issue open for now.
